.github/workflows/version_check.yml

```diff
@@ -0,0 +1,25 @@
+name: version_check
+
+on:
+  pull_request:
+    branches:
+    - '**'
+    paths:
+    # Only run when these files have been edited.
+    - 'markdown/__meta__.py'
```

Comment:
This appears to be working correctly. In [this run][1], the job was defined but did not run as no changes had been made to `markdown/__meta__.py`. And then [the job ran successfully][2] after 9ac6fd3 included a change to `markdown/__meta__.py`.

[1]: https://github.com/Python-Markdown/markdown/actions/runs/5799096176/job/15718320332
[2]: https://github.com/Python-Markdown/markdown/actions/runs/5799199515/job/15718664775?pr=1371